curriculum/challenges/english/25-front-end-development/lab-counting-cards/68a23004e96a105df3a21fc4.md

```diff
@@ -17,10 +17,12 @@ Having more high cards remaining in the deck favors the player. When the count i
 
 1. You should have a function called `cc`.
 1. The `cc` function should receive a `card` parameter which can either be a number or string. 
+1. For values between `2` to `10`, the `card` parameter will be a number. 
+1. For all other values, the `card` parameter will be a string. 
```

Comment:
I don't think these should be user stories of their own, they are specifications of the one above, you can just put it all on the same line, or use an indentend unordered list